plugin/pkg/utils/miscellaneous.go

```diff
@@ -33,7 +34,7 @@ func SkipVerification(operation admission.Operation, metadata metav1.ObjectMeta)
 }
 
 // IsSeedUsedByShoot checks whether there is a shoot cluster referencing the provided seed name
-func IsSeedUsedByShoot(seedName string, shoots []*core.Shoot) bool {
+func IsSeedUsedByShoot(seedName string, shoots []*gardencorev1beta1.Shoot) bool {
```

Comment:
I didn't expect this change after following the concept of converting to the internal version first. Is this a left over?
Similar concerns for other changes in this file, as well.